packages/accessible-react-app/src/js/components/Accordion.js

```diff
@@ -0,0 +1,52 @@
+/* https://inclusive-components.design/collapsible-sections/ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import generateRandomString from '../helpers/generateRandomString';
+
+const Accordion = props => {
+  const { accordionLabel, children } = props;
+  const [expanded, setExpanded] = useState(false);
+
+  const generatedId = generateRandomString(10);
+
+  const handleClick = eventTarget => {
+    eventTarget.preventDefault();
+    setExpanded(!expanded);
+  };
+
+  return (
+    <div>
```

Comment:
Are we doing accordions or just independent collapsible sections here? Accordions would also have keyboard bindings to the up and down arrows (and home/end) as well as focus trapping when the arrows are used. The accordion key bindings are outlined in the "Keyboard Support" section here https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html